CHANGELOG.md

```diff
@@ -1,3 +1,268 @@
+|                                          |                     |                                  |
+|                                          |                     |                                  |
+| Working Directory Endpoints              |                     |                                  |
+| "/wd/status"                             | []StatusItem        | fsi.Status                       |
+| "/wd/init"                               | DSRef               | fsi.Init                         |
+| "/wd/caninitworkdir"                     | --                  | fsi.CanInitworkdir               |
+| "/wd/checkout"                           | --                  | fsi.Checkout                     |
+| "/wd/restore"                            | --                  | fsi.Restore                      |
+| "/wd/write"                              | []StatusItem        | fsi.Write                        |
+| "/wd/createlink"                         | VersionInfo         | fsi.CreateLink                   |
+| "/wd/unlink"                             | string              | fsi.Unlink                       |
+| "/wd/ensureref"                          | --                  | fsi.EnsureRefNotLinked           |
+|                                          |                     |                                  |
+
+### "Protocol Endpoints"
+
+| Sync Endpoints                           |                     |                  |                                  |
+| ---------------------------------------- | ------------------- | ---------------- | -------------------------------- |
+| "/sync/dag"                              | GET *1              | dag.Info         | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | GET *2              | []byte           | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | POST                | dag.Manifest     | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | PUT                 | --               | remote.DsyncHTTPHandler          |
+| "/sync/dag"                              | PATCH               | stream?          | remote.DsyncHTTPHandler          |
+| "/sync/logs"                             | GET                 | --               | remote.LogbookHTTPHandlery       |
+| "/sync/logs"                             | PUT                 | []byte           | remote.LogbookHTTPHandlery       |
+| "/sync/logs"                             | DELETE              | --               | remote.LogbookHTTPHandlery       |
+| "/sync/resolve"                          | GET                 | DSRef            | remote.RefsHTTPHandler           |
+| "/sync/resolve"                          | DELETE              | --               | remote.RefsHTTPHandler           |
+|                                          |                     |                  |                                  |
```

Comment:
let's remove this section on protocol endpoints, we don't want users calling these.